tests/functions/test_select_columns.py

```diff
@@ -30,3 +30,18 @@ def test_select_columns_glob_inputs(dataframe, invert, expected):
     df = dataframe.select_columns(search_column_names=columns, invert=invert)
 
     pd.testing.assert_frame_equal(df, dataframe[expected])
+
+
+@pytest.mark.functions
+@pytest.mark.parametrize(
+    "columns",
+    [
+        ["a", "Bell__Chart", "foo"],
+        ["a", "Bell__Chart", "foo", "bar"],
+        ["a*", "Bell__Chart", "foo"],
+        ["a*", "Bell__Chart", "foo", "bar"],
+    ],
+)
+def test_select_columns_missing_columns(dataframe, columns):
```

Comment:
Can you add a (one line?) docstring to the test re: #306 to make its intent clear?

Reply:
Added one-line docstring to tests.

Reply:
Thanks @hectormz, you're a wonderful collaborator with your meticulousness :tada:

Reply:
👍 Thanks for the docstring!